content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,214 @@
+
+### Starting Deadline
+
+Bidang `.spec.startingDeadlineSeconds` adalah bidang opsional.
+Bidang tersebut berarti batas waktu dalam satuan detik untuk memulai sebuah Job jika Job melewatkan waktu yang telah dijadwalkan karena alasan apapun.
+Setelah mencapai batas waktu, CronJob tidak akan memulai sebuah Job.
+Job yang tidak memenuhi batas waktu, dengan cara ini dianggap sebagai Job yang gagal.
+Jika bidang ini tidak ditentukan, maka Job tidak memiliki batas waktu.
+
+CronJob Controller menghitung berapa banyak jadwal yang terlewat untuk sebuah CronJob. jika lebih dari 100 jadwal yang terlewat, maka tidak ada lagi CronJob yang akan dijadwalkan. Ketika `.spec.startingDeadlineSeconds` tidak disetel, CronJob Controller menghitung jadwal yang terlewat dari `status.lastScheduleTime` hingga sekarang.
+
+Misalnya, sebuah CronJob seharusnya berjalan setiap menit, `status.lastScheduleTime` CronJob adalah pukul 5:00am, tetapi sekarang pukul 07:00am. Itu berarti ada 120 jadwal yang terlewat, maka tidak ada lagi CronJob yang akan dijadwalkan.
+
+Jika bidang `.spec.startingDeadlineSeconds` disetel (tidak kosong), CronJob Controller akah menghitung berapa banyak Job yang terlewat dari nilai `.spec.startingDeadlineSeconds` hingga sekarang.
+
+Misalnya, jika disetel ke `200`, CronJob Controller akan menghitung jadwal yang terlewat dalam 200 detik terakhir. Pada kasus ini, jika terdapat lebih dari 100 jadwal yang terlewat dalam 200 detik terakhir, maka tidak ada lagi CronJob yang akan dijadwalkan. 
+
+### Kebijakan _Concurrency_
+
+Bidang `.spec.concurrencyPolicy` juga opsional.
+Bidang tersebut menentukan bagaimana memperlakukan eksekusi _concurrent_ dari sebuah Job yang dibuat oleh CronJob.
+Kamu dapat menetapkan hanya satu dari kebijakan _concurrency_:
+
+* `Allow` (bawaan): CronJob mengizinkan Job berjalan secara _concurrent_
+* `Forbid` : Job tidak mengizinkan Job berjalan secara _concurrent_; jika sudah saatnya untuk menjalankan Job baru dan Job sebelumnya belum selesai, maka CronJob akan melewatkan Job baru yang akan berjalan
+* `Replace`: Jika sudah saatnya untuk menjalankan Job baru dan Job sebelumnya belum selesai, maka CronJob akan mengganti Job yang sedang berjalan dengan Job baru.
+
+Perhatikan bahwa kebijakan _concurrency_ hanya berlaku untuk Job yang dibuat dengan CronJob yang sama.
+Jika terdapat banyak CronJob, Job akan selalu diizinkan untuk berjalan secara _concurrent_.
+
+### Penangguhan
+
+Bidang `.spec.suspend` juga opsional.
+Jika bidang tersebut disetel `true`, semua eksekusi selanjutnya akan ditangguhkan.
+Konfigurasi ini tidak dapat berlaku untuk eksekusi yang sudah dimulai.
+Secara bawaan _false_.
+
+{{< caution >}}
+Eksekusi yang ditangguhkan selama waktu yang dijadwalkan dihitung sebagai Job yang terlewat.
+Ketika `.spec.suspend` diubah dari `true` ke `false` pada CronJob yang memiliki konfigurasi tanpa [batas waktu](#starting-deadline), Job yang terlewat akan dijadwalkan segera.
+{{< /caution >}}
+
+### Job History Limits
```

Comment:
Does this term localize?
```suggestion
### _Job History_ Maksimum
```
? (I'm guessing with computer help, I don't speak bahasa Indonesia).